package.json

```diff
@@ -15,7 +15,7 @@
       "prettier --write"
     ],
     "*.py": [
-      "python3 -m pipenv run yapf --in-place"
+      "make format-python"
```

Comment:
I think the way this works is that it passes the file names for the files changed after it. That wouldn’t work so well with `make` semantics.

Reply:
Oh, gotcha. Do you suggest changing it back to an explicit `black` command?

Reply:
I went ahead and fixed it